custom_components/powercalc/data/genio/I002579/model.json

```diff
@@ -11,5 +11,9 @@
     "standby_power": 0.7,
     "supported_modes": [
         "lut"
+    ],
+    "aliases": [
+      "GENIO Smart Lighting",
```

Comment:
This seems rather generic to me, can see potential issues here when multiple lights expose the model as "GENIO Smart Lighting" to HA.

Reply:
Yeah, it's a bit silly. For some reason, that's how it gets presented from Tuya.

```
    "product_id": "7n9kwxx6zrwdqkcx",
    "product_name": "GENIO Smart Lighting",
```
I have no idea if that will conflict/cause confusion with the 3m vs 5m LED strips.